pkgs/games/portmod/tr-patcher.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, fetchzip, jre, makeWrapper }:
+let
+	translation-file = fetchurl {
```

Comment:
`s/\t/  /g`.

Reply:
I don't understand. Is that typing error or a sed script ?

Reply:
It's what ever you want :). Use whatever tool you wish, just be consistent please with spaces.

Reply:
@marius851000 That is a regex. ``\t`` is a tab and you should replace all tabs with spaces.

Reply:
Thank for this explanation. I'm not used to sed.